content/en/llm_observability/monitoring/querying.md

```diff
@@ -0,0 +1,133 @@
+
+### Query LLM input and output
+You can also search for free text in the input and output of your LLM application's spans. Wrap your free text query with `"`.
+
+For example, the query `"what's the weather"` returns spans that contain the string `what's the weather` in the input or output.
+
+<div class="alert alert-info">Free text queries are limited to the first 20,500 characters of a trace's input or output.</div>
+
+### Operators
+
+You can combine multiple search terms using the boolean operators `AND` (intersection), `OR` (union), and `-` (exclusion).
+
+For example, the query `@duration:>5s AND -"foo"` returns spans that took more than 5 seconds to complete and do **not** contain the string `foo` in the input or output.
+
+### Query syntax
+
+The LLM Observability Trace Explorer shares the same query syntax as Datadog's [APM Trace Explorer][6]. For query syntax details, including wildcard search, handling numerical values, escaping special characters, and more, see [Trace Explorer Query Syntax][6].
+
+## Example queries
+
+`@status:error`
+: Finds spans or traces that have a status of `error`.
+
+`@meta.error.type:<error_type_name>`
+: Finds spans or traces by error type. For example, `<error_type_name>` could be `"Max turns exceeded"`.
+
+`@duration:>5s`
+: Finds spans or traces that took more than 5 seconds to complete.
+
+`@trace.total_tokens:>=1000`
+: Finds traces that consumed 1000 or more total tokens.
+
+### Evaluation queries
+
+Use the `@evaluations` attribute to find spans or traces by [evaluation][3] result.
+
+`@evaluations.failure_to_answer:"failed to answer"`
+: Finds spans or traces where the LLM failed to deliver an appropriate response.
+
+`@evaluations.hallucination:"hallucination found"`
+: Finds spans or traces where a hallucination is detected.
+
+`@evaluations.input_sentiment:negative`
+: Finds spans or traces with negative input sentiment.
+
+`@evaluations.input_toxicity:toxic`
+: Finds spans or traces where harmful or inappropriate content is detected in the input.
+
+`@evaluations.language_mismatch:mismatched`
+: Finds spans or traces where the user's prompt and the LLM's response are in different languages.
+
+`@evaluations.output_sentiment:negative`
+: Finds spans or traces with negative output sentiment.
+
+`@evaluations.output_toxicity:toxic`
+: Finds spans or traces where harmful or inappropriate content is detected in the output.
+
+`@evaluations.prompt_injection:found`
+: Finds spans or traces where prompt injection is detected.
+
+`@evaluations.topic_relevancy:irrelevant`
+: Finds spans or traces where the prompt input diverges from the LLM application's intended topic.
+
+You can also search spans by the results of [custom evaluations][4]. For example, if you have a custom evaluation called `user_mood` with categorical values `happy`, `sad`, and `tired`, you could use the query: `@evaluations.custom.user_mood:happy`.
```

Comment:
can we move this to the top of this section or have it be a tab between Custom Evaluations and Datadog Evaluations?